src/main/java/com/gmail/inverseconduit/javadoc/JavaDocAccessor.java

```diff
@@ -1,130 +1,623 @@
 package com.gmail.inverseconduit.javadoc;
 
 import java.io.IOException;
-import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import com.gmail.inverseconduit.datatype.ChatMessage;
+import com.gmail.inverseconduit.utils.ChatBuilder;
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Multimap;
 
+/**
+ * Handles "javadoc" commands.
+ */
```

Comment:
Author, Implications of usage, how does it work? More information could be helpful ;)